### ISSUE TITLE
format into Forecast object

### ISSUE THREAD

**Issue by peterdudfield** [openclimatefix/neso-solar-consumer#4]
After pulling the NESO forecast, it would be good to format into OCF format. 
It would be a [Forecast](https://github.com/openclimatefix/nowcasting_datamodel/blob/main/nowcasting_datamodel/models/forecast.py#L481) object

**Comment by peterdudfield** [comment]
or we could change it straight to a ForecastSQL - a bit like here https://github.com/openclimatefix/uk-pvnet-app/blob/main/pvnet_app/forecast_compiler.py#L427

**Comment by peterdudfield** [comment]
start on this @Seao7 would be great

**Comment by Seao7** [comment]
Yeah sure! I'll get started @peterdudfield

**Comment by peterdudfield** [comment]
Great stuff, Its definately a bit more complete, with the data bases e.t.c, so please give me a shout if you need a some help setting this up

**Comment by peterdudfield** [comment]
you probably need to use `model = get_model(name=model_tag, version=version, session=session)` see [here](https://github.com/openclimatefix/uk-pvnet-app/blob/main/pvnet_app/forecast_compiler.py#L382C9-L382C76) and `location = get_location(session=session, gsp_id=0))` like [here](https://github.com/openclimatefix/uk-pvnet-app/blob/main/pvnet_app/forecast_compiler.py#L391C13-L391C73)

**Comment by siddharth7113** [comment]
Hi @Seao7 and @peterdudfield,

I noticed there hasn’t been much activity on Issue #4 recently. If there’s no ongoing work, I’d love to give it a try. Please let me know if that works!

**Comment by peterdudfield** [comment]
Yea go for it, is that ok @Seao7 ?

**Comment by Seao7** [comment]
Yeah please go ahead, have been occupied for the past week @peterdudfield !!
